index.js

```diff
@@ -744,6 +744,81 @@ class WebUntis {
         if (response.data.result.code) throw new Error('Server returned error code: ' + response.data.result.code);
         return response.data.result;
     }
+
+    /**
+     * Returns all the Lessons where you were absent including the excused one!
+     * @param {Date} rangeStart
+     * @param {Date} rangeEnd
+     * @param {Integer} execuseStatusId
+     * @param {boolean} [validateSession=true]
+     * @returns {Promise.<Array>}
+     */
+    async getAbsentLesson(validateSession = true, rangeStart, rangeEnd, excuseStatusId = -1) {
```

Comment:
Sorry for only noticing now, but you forgot to add type definitions (index.d.ts) for the new function.